test/unittests/evm_fixture.hpp

```diff
@@ -35,46 +35,55 @@
 class evm : public testing::Test
 {
 protected:
+    /// The VM handle.
     evmc::vm& vm;
-    evmc_revision rev = EVMC_BYZANTIUM;  // Byzantium by default. TODO: Add alias evmc::revision.
-    evmc_message msg = {};               // TODO: Add evmc::message with default constructor.
-    evmc::result result{{}};  // TODO: Add default constructor to evmc::result, update code here.
+
+    /// The EVM revision for unit tests execution. Byzantium by default.
+    /// TODO: Add alias evmc::revision.
+    evmc_revision rev = EVMC_BYZANTIUM;
+
+    /// The message to be executed by an unit test (with execute() method).
+    /// TODO: Add evmc::message with default constructor.
+    evmc_message msg{};
+
+    /// The result of execution (available after execute() is invoked).
+    /// TODO: Add default constructor to evmc::result, update code here.
+    evmc::result result{{}};
+
+    /// The total amount of gas used during execution.
     int64_t gas_used = 0;
 
     MockedHost host;
 
     evm() noexcept : vm{get_vm()} {}
 
-    /// Wrapper for evmone::execute. The result will be in the .result field.
-    void execute(int64_t gas, bytes_view code, std::string_view input_hex = {}) noexcept
+
+    /// Executes given code.
```

Comment:
(Nit) Executes given code -> executes the supplied code (or “executes the given code”)